app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -36,6 +36,7 @@ class << model_name
 
     validates_presence_of :zone
 
+    include StorageManagerMixin
```

Comment:
I think the StorageManagerMixin should be included in the openstack/cloud_manager.rb, not in the top-level cloud_manager.rb for the same reason as previously discussed.  The Mixin file defines a cinder_manager (and a swift_manager for me) - and that should not happen for other cloud_managers such as Amazon etc. unless specifically defined for that cloud manager like OpenStack.